src/DataSource.ts

```diff
@@ -0,0 +1,26 @@
+import { DataSourceInstanceSettings, MetricFindValue } from '@grafana/data';
+import { DataSourceWithBackend } from '@grafana/runtime';
+
+import { InfluxQuery, InfluxOptions } from './types';
+
+export class DataSource extends DataSourceWithBackend<InfluxQuery, InfluxOptions> {
+  /** @ngInject */
```

Comment:
Is ngInject still for the template server? I thought this had been converted away from angular?